### DISCUSSION TITLE
Feature Request: Directory Structure

### DISCUSSION THREAD

**Discussion by rhaussmann** [gotson/komga#1148] [Ideas]
Comicrack refugee here. I'd like to request an option for a directory-like view.

I really miss having a directory-like interface structure with columns for metadata fields with functionality for sorting and grouping. 
I miss the ability to compare files by data field. Normally I would "prune(delete)" files from the library that may be duplicated, uninteresting, or have been collected (and are a kind of duplication). 

This might also serve as a solution for the delay caused by generating thumbnails in the current interface. 

YACReader also has this interface, though the fields can't be sorted and the import for the metadata import isn't quite ready yet. (This is based on the last time I used it, which was some time ago. The dev is a great guy and I still use the YACReader.) 

Thanks for the great work!

**Comment by gotson** [comment]
It's been requested in the past (https://github.com/gotson/komga/issues/401), but isn't likely to happen at all. The metadata model is far more superior than a arbitrary folder structure.

**Reply by stanthewizzard** [reply]
yes BUT ... it's easier to remind of the folder from the nas :)
